tests/codegen/checked_ilog.rs

```diff
@@ -5,7 +5,7 @@
 // Ensure that when val < base, we do not divide or multiply.
 
 // CHECK-LABEL: @checked_ilog
-// CHECK-SAME: (i16 noundef %val, i16 noundef %base)
+// CHECK-SAME: (i16{{( .+)?}} %val, i16{{( .+)?}} %base)
```

Comment:
```suggestion
// CHECK-SAME: (i16{{.*}} %val, i16{{.*}} %base)
```
Should be enough, there's no need to explicitly match the space, we're not really afraid of accidentally generating i165 here...

Reply:
Makes sense, this is more clear indeed. Done.